lib/dynflow/actors/execution_plan_cleaner.rb

```diff
@@ -0,0 +1,52 @@
+module Dynflow
+  module Actors
+    class ExecutionPlanCleaner
+      attr_reader :core
+
+      def initialize(world, options = {})
+        @world = world
+        @options = options
+      end
+
+      def core_class
+        Core
+      end
+
+      def spawn
+        Concurrent.future.tap do |initialized|
+          @core = core_class.spawn(:name => 'execution-plan-cleaner',
+                                   :args => [@world, @options],
+                                   :initialized => initialized)
+        end
+      end
+
+      def clean!
+        core.tell([:clean!])
+      end
+
+      class Core < Actor
+        def initialize(world, options = {})
+          @world = world
+          default_age = 60 * 60 * 24 # One day by default
+          @poll_interval = options.fetch(:poll_interval, default_age)
+          @max_age = options.fetch(:max_age, default_age)
+          start
+        end
+
+        def start
+          set_clock
+          clean!
+        end
+
+        def clean!
+          plans = @world.persistence.find_old_execution_plans(Time.now.utc - @max_age)
+          @world.persistence.delete_execution_plans(uuid: plans.map(&:id))
```

Comment:
Doesn't this have the issue with the blank filer on `uuid => []` matching all the plans? I guess we should probably just fix this case directly in sequel persistence adapter, to avoid this issues in once and for all, as I don't see where it would actually be useful.